test/token/ERC721/extensions/ERC721Consecutive.t.sol

```diff
@@ -119,4 +152,29 @@ contract ERC721ConsecutiveTest is Test {
         assertEq(token.balanceOf(accounts[0]), batches[0]);
         assertEq(token.balanceOf(accounts[1]), batches[1]);
     }
+
+    function test_start_consecutive_id(
+        address receiver,
+        uint256[2] calldata unboundedBatches,
+        uint256[2] calldata unboundedTokenId,
+        uint96 startingId
+    ) public {
+        vm.assume(receiver != address(0));
+
+        uint256 startingTokenId = bound(startingId, 1, 5000);
+
+        // We assume _maxBatchSize is 5000 (the default). This test will break otherwise.
```

Comment:
Why is this breaking? If it's breaking because of the `ownerOf` range check, we definitely need to address that.

Reply:
The comment about the breaking test is from a previous commit which I still can't decipher

Reply:
The test tries to mint a batch that is too large. too large means bigger than `_maxBatchSize`. Since the contract is not yet constructed, we assume the value is 5000. If you ever override `_maxBatchSize` in `ERC721ConsecutiveTarget` , you should make sure it is reflected here.

Reply:
Ah, understood. Thanks for clarifying!